installation/scripts/testing.sh

```diff
@@ -11,7 +11,7 @@ function validateConcurrency() {
     exit 1
   fi
 
-  if ! [[ "$1" =~ '^[0-9]+$' ]]; then
+  if ! [[ "$1" =~ ^[0-9]+$ ]]; then
```

Comment:
Excuse me for being a pain in the neck, but as far as I can tell, this regexp matches leading zeros.

Reply:
Yes, it does... Gimme a sec. I'll teach him good manners.